src/entry-point-static/build.tsx

```diff
@@ -0,0 +1,24 @@
+import fs from "fs";
+import path from "path";
+
+import generatePageContent, { PageInfo } from "./generatePageContent";
+
+let pageInfos: PageInfo[] = [
+  { title: "Recidiviz COVID-19 Dashboard", location: "/" },
+  { title: "Test Page", location: "/test" },
+];
+
+console.log();
```

Comment:
I think you forgot about this one?

Reply:
thanks for the callout!

this line is intentional -- it prints the blank line highlighted here, just to create some space between the command outputs:

![image](https://user-images.githubusercontent.com/1570168/77835910-ba760900-710e-11ea-883a-8d544a93703a.png)

It does feel weird to put it here though. Moving to a more reasonable place:

In `package.json`:

```diff
    "scripts": {
-     "build": "parcel build index.html &&         parcel build --target node src/entry-point-static/build.tsx &&         node dist/build.js && rm -f dist/build.js dist/build.js.map"
+     "build": "parcel build index.html && echo && parcel build --target node src/entry-point-static/build.tsx && echo && node dist/build.js && rm -f dist/build.js dist/build.js.map"
    }
```